src/furo/assets/styles/content/_lists.sass

```
@@ -19,6 +19,18 @@ ol
       margin-top: 0.5rem
       margin-bottom: 0.5rem
 
+ol
+  .arabic
+    list-style: decimal
+  .loweralpha
+    list-style: lower-alpha
+  .upperalpha
+    list-style: upper-alpha
+  .lowerroman
+    list-style: lower-roman
+  .upperroman
+    list-style: upper-roman
```

```suggestion
  &.arabic
    list-style: decimal
  &.loweralpha
    list-style: lower-alpha
  &.upperalpha
    list-style: upper-alpha
  &.lowerroman
    list-style: lower-roman
  &.upperroman
    list-style: upper-roman
```

These need ampersands, otherwise the selector becomes `ol .arabic` instead of `ol.arabic`.

---

Right you are!